lib/cli/ui/glyph.rb

```diff
@@ -38,20 +38,14 @@ def initialize(handle, codepoint, color)
 
       # Mapping of glyphs to terminal output
       MAP = {}
-      # YELLOw SMALL STAR (⭑)
-      STAR     = new('*', 0x2b51,  Color::YELLOW)
-      # BLUE MATHEMATICAL SCRIPT SMALL i (𝒾)
-      INFO     = new('i', 0x1d4be, Color::BLUE)
-      # BLUE QUESTION MARK (?)
-      QUESTION = new('?', 0x003f,  Color::BLUE)
-      # GREEN CHECK MARK (✓)
-      CHECK    = new('v', 0x2713,  Color::GREEN)
-      # RED BALLOT X (✗)
-      X        = new('x', 0x2717,  Color::RED)
-      # Bug emoji (🐛)
-      BUG      = new('b', 0x1f41b, Color::WHITE)
-      # RIGHT-POINTING DOUBLE ANGLE QUOTATION MARK (»)
-      CHEVRON  = new('>', 0xbb,    Color::YELLOW)
+      STAR      = new('*', 0x2b51,           Color::YELLOW) # YELLOw SMALL STAR (⭑)
```

Comment:
```suggestion
      STAR      = new('*', 0x2b51,           Color::YELLOW) # YELLOW SMALL STAR (⭑)
```
Might as well change it